### PR TITLE
Feat/edit account

### PR DESCRIPTION
**Pull request by drikusroor**
This PR will allow you to change your name on the edit account page. You can reach it by going to the dashboard, then to account, and click edit. Edit your name and click save. Or just go to `/admin/account/edit`. The updated name will also show under your posts. We can later use this setup as inspiration for editing other properties on entities, perhaps when we want to add the bio to our user profiles or something instead of hard-coding it like we do now.

### CODE REVIEW

**Comment by naomi-eliasar** [comment]
I am not sure if it has to do with this branch, but I get an error on the blog page and an error on log in, so I cannot reach the admin dashboard and changes in this PR
login error
No blog

**Comment by drikusroor** [comment]
@naomi-eliasar can you confirm it's working again?

File: web/src/components/Article/Article.tsx [hunk 1]
```diff
@@ -7,23 +7,29 @@ interface Props {
...
-          <span className="text-sm text-gray-400">
+        <div className="flex flex-row items-end gap-2">
+          <span className="text-sm text-slate-500">
             {article.user.name
-              ? `by ${article.user.name}`
+              ? article.user.name
               : article.user.email
               ? article.user.email
-              : 'by Anonymous'}
+              : 'Anonymous'}
```
Comment (adrianabm) [inline web/src/components/Article/Article.tsx:30]: huh? ternary nesting is so confusing 😝

**Review by adrianabm — APPROVED** [review]
LGTM!!!!!!!

File: web/src/components/Article/Article.tsx (hunk 1 above)
Comment (drikusroor) [reply web/src/components/Article/Article.tsx:30]: Is it? 😮

**Pull request merged**